freqtrade/persistence/models.py

```diff
@@ -282,6 +292,12 @@ def close_date_utc(self):
         return self.close_date.replace(tzinfo=timezone.utc)
 
     def to_json(self) -> Dict[str, Any]:
+        fill_buy = self.select_filled_orders('buy')
+        buys_json = dict()
+        if len(fill_buy) > 0:
+            for x in range(len(fill_buy)):
+                buys_json[str(x)] = fill_buy[x].to_json()
```

Comment:
Reminder to self: 
Investigate impact on backtesting filesize - as this will be exported there, too.

Reply:
Don't think this is relevant for backtesting - backtesting only writes "selected" columns to file.